src/redturtle/prenotazioni/restapi/serializers/adapters/booking_type.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from zope.component import adapter
+from zope.interface import implementer
+
+from redturtle.prenotazioni.content.booking_type import IBookingType
+from redturtle.prenotazioni.interfaces import (
+    IRedturtlePrenotazioniLayer,
+    ISerializeToRetrocompattibleJson,
+)
+
+
+@implementer(ISerializeToRetrocompattibleJson)
+@adapter(IBookingType, IRedturtlePrenotazioniLayer)
+class BookingTypeRetrocompattibleSerializer:
```

Comment:
add a comment to explain why we are using this serializer.
I would not call it retrocompatible because is basically a simplified serialization

Reply:
Added  a docstring. It just must be a short reminder that it is only a temporary solution.